src/main/java/org/stellar/sdk/responses/operations/InvokeHostFunctionOperationResponse.java

```diff
@@ -0,0 +1,152 @@
+package org.stellar.sdk.responses.operations;
+
+import com.google.common.collect.ImmutableList;
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * Represents InvokeHostFunction operation response.
+ *
+ * @see <a href="https://developers.stellar.org/api/horizon/resources/operations"
+ *     target="_blank">Operation documentation</a>
+ * @see org.stellar.sdk.requests.OperationsRequestBuilder
+ * @see org.stellar.sdk.Server#operations()
+ */
+public class InvokeHostFunctionOperationResponse extends OperationResponse {
+  @SerializedName("function")
+  private final String function;
```

Comment:
might want to call this `functionType` because the value from api will be one of these enum names - https://github.com/stellar/go/blob/soroban-xdr-next/xdr/xdr_generated.go#L24607-L24612

Reply:
I prefer to maintain consistency with [the definition in Horizon](https://github.com/stellar/go/blob/soroban-xdr-next/protocols/horizon/operations/main.go#L362). Perhaps we should modify the definition in Horizon to functionType?

Reply:
ok, good point, for now in sdk, how about a comment in code/javadocs that mentions the possible enum'd values from https://github.com/stellar/go/blob/soroban-xdr-next/xdr/xdr_generated.go#L24607-L24612 or a unit test that shows the values? that reference may help developers reference this field correctly.

Reply:
Fixed in 81d23c5ec221d3ad2d1471c6a6f2a3ee20152a6c.